DevFiles/rajendra.txt

```diff
@@ -1,2 +1 @@
-this is rajendra.
-welcome to ssi.
+this is rajendra.welcome to ssi.
```

Comment:
Make the first letter of the line as Capital

Reply:
i changed to cap letter.